src/main/scripts/informatica_deploy_deployment.groovy

```diff
@@ -92,13 +94,23 @@ println('')
 
 def script = new File(inputFile)
 script << "connect -r $srcrepo -n $srcusername -x $srcpassword "
+
+if (srcsecurityDomain){
```

Comment:
Do both of these properties use the -s flag? Are they supposed to? 

If they do, what happens if they are both set? Does the second override the other?

Reply:
The command looks like something like :
connect -r ... -n ... -x ... -s ... deploydeploymentgroup -p ... -c ... -r ... -n ... -x ... -s ... 
you can see that most flags are duplicated.

Looking at some doc of deploydeploymentgroup command:

pmrep DeployDeploymentGroup
The DeployDeploymentGroup command uses the following syntax:
deploydeploymentgroup
-p <deployment_group_name>
-c <control_file_name>
-r <target_repository_name>
[-n <target_repository_user_name>
[-s <target_repository_user_security_domain>]
 [-x <target_repository_password> |
 -X <target_repository_password_environment_variable>]
 [-d <target_domain_name> |
 {-h <target_portal_host_name>
 -o <target_portal_port_number>}]] (only if target is in a differentdomain)
[-l <log_file_name>]

link:
https://kb.informatica.com/h2l/HowTo%20Library/1/0083-DeploymentUsingPmrepCommands.pdf

Reply:
Gotcha thanks!